Examples/SampleAppSwift/SampleAppSwift/AppDelegate.swift

```diff
@@ -18,8 +18,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
         
-        let config: RSConfig = RSConfig(writeKey: "1wvsoF3Kx2SczQNlx1dvcqW9ODW")
-            .dataPlaneURL("https://rudderstacz.dataplane.rudderstack.com")
+        let config: RSConfig = RSConfig(writeKey: "")
+            .dataPlaneURL("")
```

Comment:
Lets add a placeholder:
```suggestion
        let config: RSConfig = RSConfig(writeKey: "<WRITE_KEY>")
            .dataPlaneURL("<DATA_PLANE_URL>")
```

Reply:
Updated hint for key and url.